src/app/components/Login/login.tsx

```diff
@@ -1,19 +1,32 @@
-import { FormEvent, useState } from "react";
+import { FormEvent, useEffect, useState } from "react";
 import styles from "./login.module.css";
 
 export default function Login(): JSX.Element {
-  const [name, setName] = useState("");
+  const [name, setName] = useState(
+    sessionStorage.getItem("Current User") || ""
```

Comment:
Just a quick note (that doesn't need to be applied): 
If you're saving the user in the session storage, it will be deleted after the user closes the session and will have to login again for the next session. If that's what you want here, that's totally fine. 

An alternative you could use is locale storage here or even cookies. With cookies you could later fetch the user with `/api/users/me`.